demo/chatbot_retry_undo_like/run.py

```diff
@@ -22,7 +22,7 @@ def respond(
         stream=True,
         model="HuggingFaceH4/zephyr-7b-beta"
     ):
-        response["content"] += message.choices[0].delta.content or ""
+        response["content"] += message.choices[0].delta.content or "" if message.choices else ""
```

Comment:
We may also want to check `if len(message.choices)`. The other day I got an error because it was an empty list

Reply:
An empty list is falsey, so `if message.choices` covers that case too.

```python
>>> a = []
>>> bool(a)
False
```

You're correct that the old code would crash.